src/Icon/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import styles from './Icon.scss';
+
+function Icon(props) {
+  return (
+    <div>
+      <span
+        id={props.id}
+        className={classNames(props.className)}
+        aria-hidden={props.hidden}
+      />
+      { props.screenReaderText &&
+        <span className={classNames(styles['sr-only'])}>
+          {props.screenReaderText}
+        </span>
+      }
+    </div>
+  );
+}
+
+Icon.propTypes = {
+  id: PropTypes.string.isRequired,
```

Comment:
Why is `id` required? Could we leverage `utils/newId` instead?

Reply:
Good spot -- It doesn't need to be required, I think I just did this out of habit and didn't realize -- will update to use the newId as we do elsewhere.